### ISSUE TITLE
FreeBSD Port 

### ISSUE THREAD

**Issue by lbartoletti** [sqlitebrowser/sqlitebrowser#180]
Hi,

I am pleased to announce that it's done! I created the ports and it was commited.
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=196365

You can install it via FreeBSD ports system:
#make -C /usr/ports/databases/sqlitebrowser install

**Comment by lbartoletti** [comment]
Yes,

You have two choice for install it:
Using port:
#make -C /usr/ports/databases/sqlitebrowser install

Using package:
#pkg install quiterss

Regards

**Comment by justinclift** [comment]
Is "quiterss" the correct package name there?

**Comment by lbartoletti** [comment]
Sorry...

I works on quiterrss package too..
Remove pkg install quiterss.
It's:
#pkg install sqlitebrowser

**Comment by justinclift** [comment]
Thanks, updated the installation info. :smile:

Btw, we'll probably be releasing v3.5.0 in the next few days.  Would you be ok to update the FreeBSD port when that happens?

**Comment by lbartoletti** [comment]
Yes, I will update the port.

**Comment by justinclift** [comment]
Thanks.  It'll probably be this weekend. :smile:

**Comment by justinclift** [comment]
@lbartoletti 3.5.0 is out now. :smile:

**Comment by lbartoletti** [comment]
Done with a patch for translations: using Datadir (/usr/local/share/sqlitebrowser/) when installing instead of binary dir (/usr/local/bin).

Waiting for approval https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=197253

**Comment by justinclift** [comment]
Thanks, that's awesome. :grinning:

**Comment by justinclift** [comment]
Out of curiosity, the port seems to be carrying some patches (as below), should we merge these into the main codebase?

```
+--- src/Application.cpp.orig	2015-01-30 19:48:06.000000000 +0100
++++ src/Application.cpp	2015-02-01 15:40:02.790619098 +0100
+@@ -30,8 +30,15 @@
+ 
+     // First of all try to load the application translation file.
+     m_translatorApp = new QTranslator(this);
++
++#ifdef Q_OS_FREEBSD
++    QString trans_dir = "%%DATADIR%%/translations";
++    ok = m_translatorApp->load("sqlb_" + name,
++                               trans_dir);
++#else
+     ok = m_translatorApp->load("sqlb_" + name,
+                                QCoreApplication::applicationDirPath() + "/translations");
++#endif
+ 
+     if (ok == true) {
+         PreferencesDialog::setSettingsValue("General", "language", name);
+@@ -47,7 +54,9 @@
+         ok = m_translatorQt->load("qt_" + name,
+                                   QLibraryInfo::location(QLibraryInfo::TranslationsPath));
+         if (ok == false)
++	{
+             ok = m_translatorQt->load("qt_" + name, "translations");
++	}
+         if (ok == true)
+             installTranslator(m_translatorQt);
+     }
```

```
+--- src/PreferencesDialog.cpp.orig	2015-02-01 15:27:53.407669984 +0100
++++ src/PreferencesDialog.cpp	2015-02-01 15:30:51.885904399 +0100
+@@ -266,9 +266,13 @@
+ void PreferencesDialog::fillLanguageBox()
+ {
+     // Use the path relative to the main executable
++#ifdef Q_OS_FREEBSD
++    QDir translationsDir("%%DATADIR%%/translations",
++                         "sqlb_*.qm");
++#else
+     QDir translationsDir(QCoreApplication::applicationDirPath() + "/translations",
+                          "sqlb_*.qm");
+-
++#endif
+     // Add default language
+     ui->languageComboBox->addItem("English (United States)", "en_US");
```

**Comment by justinclift** [comment]
@lbartoletti It turns out there were a few important bugs in 3.5.0, so v3.5.1 has just been released (if you want to update the port?). :smile:

**Comment by lbartoletti** [comment]
Hello, I do not think that this patch is to be applied in the code base, it was necessary for me to make a clean install on FreeBSD. Nevertheless, it may be possible to perform a modification of the code to be compatible with all systems. Indeed, SqliteBrowser search the directory "translations" where it is installed (local dir or install dir /usr/[local]/bin). 
Now it is in my view a mistake when it is installed, it must be in the directory of application-related documents (/usr/[local]/share). If the patch is to be applied, it should probably create a variable in CMake replacing %%DATADIR%% the right path (/usr/share/sqlitebrowser, /usr/local/share/ sqlitebrowser/, ./ etc.)
What did you think?

**Comment by justinclift** [comment]
@MKleusberg Hmmm, any ideas? :smile:

**Comment by lbartoletti** [comment]
sqlitebrowser 3.5.1 commited on FreeBSD :)
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=197474

**Comment by justinclift** [comment]
Thank you. :grinning:

**Comment by justinclift** [comment]
@lbartoletti We've just released v3.6.0.  Would you be ok to update the port with it? :smile:
